File: paris-test/src/test/resources/styleable_minimal/input/MyView.java
package com.airbnb.paris.test;

import android.content.Context;
import android.util.AttributeSet;
import android.view.View;

import com.airbnb.paris.annotations.Styleable;

@Styleable
public class MyView extends View {

    public MyView(Context context) {
        super(context);
    }

    public MyView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public MyView(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
    }
}

File: paris-test/src/test/resources/attrs_r_class_import_fully_qualified/input/MyView.kt
package com.airbnb.paris.test

import android.content.Context
import android.util.AttributeSet
import android.view.View
import androidx.annotation.ColorInt
import com.airbnb.paris.annotations.Attr
import com.airbnb.paris.annotations.Style
import com.airbnb.paris.annotations.Styleable
import com.airbnb.paris.extensions.myViewStyle
import com.airbnb.paris.test.R2.styleable.Formats_formatBoolean

@Styleable("Formats")
class MyView @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyle: Int = 0
) : View(context, attrs, defStyle) {

    // Makes the method public
    @Attr(Formats_formatBoolean)
     fun showDivider(show: Boolean)  {

    }
}